### ISSUE TITLE
Update integrated gradients

### ISSUE THREAD

**Issue by chihyanghsu0805** [chihyanghsu0805/machine_learning#12]
https://keras.io/examples/vision/integrated_gradients/

**Comment by chihyanghsu0805** [comment]
Update with tf version, https://www.tensorflow.org/tutorials/interpretability/integrated_gradients
